suite-native/module-send/src/sendOutputsFormSchema.ts

```diff
@@ -194,10 +194,36 @@ export const sendOutputsFormValidationSchema = yup.object({
             }),
         )
         .required(),
+    enableDestinationTagValidation: yup.boolean(),
     rippleDestinationTag: yup
         .string()
-        .optional()
+        .when('enableDestinationTagValidation', {
+            is: true,
+            then: schema => schema.required('Destination Tag is required'),
+            otherwise: schema => schema.notRequired(),
+        })
         .matches(/^\d*$/, 'You can only use positive numbers for the destination tag.')
+        .test(
+            'is-destination-tag-set',
```

Comment:
nit: I would change name of this validation to this:
```suggestion
            'is-destination-tag-required',
```

Because it runs even for non XRP coins where it is not required.

Reply:
d44a49161a